src/activities/photo_hunter/resource/README

```diff
@@ -0,0 +1,11 @@
+https://openclipart.org/detail/17626/castle
```

Comment:
We need to keep track of the reference on each resource file. For example we need to keep track of what the file photo1.svg refers to.